docs/root/manual/getting-started/run-signer-node.md

```diff
@@ -424,10 +472,58 @@ sudo iptables -L -v
 sudo service netfilter-persistent save
 ```
 
+:::tip
+Here is a example of the command to run :
+
+```bash
+sudo iptables -A INPUT -s 123.123.123.123 -p tcp --dport 3128 -j ACCEPT
+sudo iptables -L -v
+sudo service netfilter-persistent save
+```
+:::
+
 ## Verify the Mithril Signer deployment
 
 :::tip
+You can verify that your signer node is correctly configured and registered by checking your signer node logs or by running this command :
+
+```bash
+if grep -q "STATE MACHINE: new cycle: Registered" **YOUR_SIGNER_NODE_LOGS_PATH**; then
+    echo ">> Your signer node is registered"
+else
+    echo ">> Your signer node is not registered"
+fi
+```
+
 There is a delay of `2` epochs between the registration of the signer node and its ability to generate individual signatures. This delay is further explained in the [Mithril certificate chain in depth](https://mithril.network/doc/mithril/mithril-protocol/certificates) documentation.
 
-Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer).
+Once this delay has passed, you should be able to observe your `PoolId` listed in some of the certificates accessible on the [`Mithril Explorer`](https://mithril.network/explorer) or by running the following command : 
+
+```bash
+party_id=**YOUR_POOL_ID**
+aggregator_endpoint=**YOUR_AGGREGATOR_ENDPOINT**
+found=false
+
+curl -s "$aggregator_endpoint/certificates" -H 'accept: application/json' | jq -r '.[] | .hash' | while read -r hash; do
+    response=$(curl -s "$aggregator_endpoint/certificate/$hash" -H 'accept: application/json')
+    signer_count=$(echo "$response" | jq '.metadata.signers | length')
+    for (( i=0; i < signer_count; i++ )); do
+        party_id_response=$(echo "$response" | jq -r ".metadata.signers[$i].party_id")
+        if [[ "$party_id_response" == "$party_id" ]]; then
+            echo ">> You have signed this certificate : $hash"
+            found=true
+            break
+        fi
+    done
+
+    if [ "$found" = true ]; then
+        break
+    fi
+done
+
+if [ "$found" = false ]; then
+    echo ">> Your party id was not found in the last certificates"
+fi
```

Comment:
Same here, create a file and explain how to run it.
Alos the env vars should be capitalized